english/challenge-marshmellow-to-rock-off/light-route/joker.md

```diff
@@ -0,0 +1,22 @@
+It is a difficult choice but in the end you take heed of the "Keep out" 
+warning and rush towards the dim light. As you pace yourself making sure 
+that no marshmallows had followed you, your eyes get adjusted to the 
+darkness and you realize that it was an elevator's light blinking 
+furiously, the number "3". You creep towards the elevator to realize 
+that it has been opening and closing repeatedly due to the doors not 
+being able to close completely. 
+
+A tiny ring is in the path of the steel doors of the elevator. 
+You pick it up and walk inside the otherwise ordinary looking lift. 
+As you try the emergency phone button without much success, 
+you realize that some of the buttons on the elevator are missing 
+and there is a tiny camera looking at you from above.
+
+Who is on the other side, you cannot tell; but, for now, what will you do?
+
+(Basement)[basement-route/cellar.md]
```

Comment:
Did you mean for this to be a link?  If so, your syntax is backwards.  The text should be in square brackets and the link in parentheses rather than vice versa.  You can view the page on GitHub on your fork to check if the link is displaying correctly.